build/DependencyVersions.props

```diff
@@ -40,10 +40,10 @@
     <MicrosoftDotNetPlatformAbstractionsPackageVersion>2.1.0</MicrosoftDotNetPlatformAbstractionsPackageVersion>
     <MicrosoftExtensionsDependencyModelPackageVersion>2.1.0</MicrosoftExtensionsDependencyModelPackageVersion>
     <MicrosoftDotNetCliCommandLinePackageVersion>0.1.1</MicrosoftDotNetCliCommandLinePackageVersion>
-    <MicrosoftDotNetProjectJsonMigrationPackageVersion>1.3.1</MicrosoftDotNetProjectJsonMigrationPackageVersion>
+    <MicrosoftDotNetProjectJsonMigrationPackageVersion>1.3.1-preview.19067.1</MicrosoftDotNetProjectJsonMigrationPackageVersion>
```

Comment:
This probably should have been bumped to 1.3.2. This version looks older. It doesn't really matter though what the version of this is I guess.

Reply:
Actually at some point a stabilized build will get kicked off again potentially and become 1.3.1 duplicate. We should bump it, but that can happen after this I think.

Reply:
@nguerrera  good point. Could you approve this branding update https://github.com/dotnet/cli-migrate/pull/109

Reply:
Done. Thanks.